### ISSUE TITLE
Quick Example errors

### ISSUE THREAD

**Issue by marcobd3** [ISG-Siegen/Auto-Surprise#49]
Hello,
I was trying to run the Quick Example (https://auto-surprise.readthedocs.io/en/stable/usage/quick_start.html#quick-example ) but I got this output:

```
auto_surprise 0.1.8
Available CPUs: 16
Evaluating RMSE, MAE, MSE of algorithm NormalPredictor on 5 split(s).

                  Fold 1  Fold 2  Fold 3  Fold 4  Fold 5  Mean    Std     
RMSE (testset)    1.5145  1.5229  1.5190  1.5152  1.5235  1.5190  0.0037  
MAE (testset)     1.2179  1.2228  1.2170  1.2117  1.2180  1.2175  0.0035  
MSE (testset)     2.2937  2.3193  2.3074  2.2960  2.3210  2.3075  0.0114  
Fit time          0.06    0.06    0.07    0.07    0.07    0.07    0.01    
Test time         0.08    0.04    0.07    0.08    0.08    0.07    0.01    
Baseline loss: 1.5190366726386277
Starting process with svd algorithm
Starting process with svdpp algorithm
Starting process with nmf algorithm
Starting process with knn_basic algorithm
Starting process with knn_baseline algorithm
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo svd
Starting process with knn_with_means algorithm
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]Exception for algo svdpp
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Starting process with knn_with_z_score algorithm
Exception for algo nmf
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

Starting process with co_clustering algorithm
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo knn_basic
Starting process with slope_one algorithm
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

Starting process with baseline_only algorithm
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]Exception for algo knn_baseline
  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo knn_with_means
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo knn_with_z_score
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo co_clustering
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

  0%|                                                                                                                                                              | 0/100 [00:00<?, ?trial/s, best loss=?]
Exception for algo baseline_only
Traceback (most recent call last):
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/trainer.py", line 85, in start_with_limits
    _, best_trial = self.algo_base.best_hyperparams(max_evals)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/auto_surprise/algorithms/base.py", line 99, in best_hyperparams
    best = fmin(**fmin_args)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 540, in fmin
    return trials.fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/base.py", line 671, in fmin
    return fmin(
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 586, in fmin
    rval.exhaust()
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 364, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.asynchronous)
  File "/home/mbaleato/AutoSurpriseExample/venv/lib/python3.8/site-packages/hyperopt/fmin.py", line 279, in run
    new_ids, self.domain, trials, self.rstate.integers(2 ** 31 - 1)
AttributeError: module 'numpy.random' has no attribute 'integers'

Evaluating RMSE, MAE, MSE of algorithm SlopeOne on 5 split(s).

                  Fold 1  Fold 2  Fold 3  Fold 4  Fold 5  Mean    Std     
RMSE (testset)    0.9453  0.9533  0.9335  0.9422  0.9494  0.9447  0.0068  
MAE (testset)     0.7435  0.7469  0.7337  0.7397  0.7495  0.7426  0.0056  
MSE (testset)     0.8936  0.9089  0.8713  0.8878  0.9014  0.8926  0.0128  
Fit time          0.39    0.36    0.34    0.40    0.38    0.38    0.02    
Test time         1.59    1.20    1.30    1.50    1.20    1.36    0.16    
----Done!----
Best algorithm: slope_one
Best hyperparameters: None
┏━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━┓
┃ Algorithm        ┃ Hyperparameters ┃               Loss ┃
┡━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━┩
│ svd              │ None            │               None │
│ svdpp            │ None            │               None │
│ nmf              │ None            │               None │
│ knn_basic        │ None            │               None │
│ knn_baseline     │ None            │               None │
│ knn_with_z_score │ None            │               None │
│ knn_with_means   │ None            │               None │
│ co_clustering    │ None            │               None │
│ baseline_only    │ None            │               None │
│ slope_one        │ None            │ 0.9447460038754011 │
└──────────────────┴─────────────────┴────────────────────┘
```

I have tried with other dataset, and always the same AttributeError: 
`AttributeError: module 'numpy.random' has no attribute 'integers'`

Is there another version that works fine? Or another version of numpy?

**Comment by thededlier** [comment]
Hey @marcobd3 did you try with numpy version 1.22.3 ?

**Comment by marcobd3** [comment]
Hey,
I have tried numpy==1.22.3 and hyperopt==0.2.7 with the same result.

**Comment by thededlier** [comment]
Try setting your random state when initializing the engine like this 

`engine = Engine(verbose=True, random_state=numpy.random.default_rng())`

The error you are seeing is because Auto-Surprise by default is using the now deprecated `numpy.random.RandomState`.

**Comment by thededlier** [comment]
Fixed in #50 and released in v0.1.9
